src/compiler/crystal/tools/doc/method.cr

```diff
@@ -135,6 +135,16 @@ class Crystal::Doc::Method
     end
   end
 
+  def visibility
+    case @def.visibility
+    in .public?
+    in .protected?
+      "protected "
+    in .private?
+      "private "
+    end
```

Comment:
Adding a space should the template's job (or helpers). The object should just return the value (`doc/macro.cr` is doing that, but `doc/type.cr` is also adding a space).